crates/matrix-sdk-crypto/src/backups/keys/decryption.rs

```diff
@@ -383,4 +397,16 @@ mod tests {
             .decrypt_session_data(encrypted.session_data)
             .expect("We should be able to decrypt a just encrypted room key");
     }
+
+    #[test]
+    fn key_matches() {
+        let decryption_key = BackupDecryptionKey::new().unwrap();
+
+        let key_info = decryption_key.to_backup_info();
+
+        assert!(
+            decryption_key.backup_key_matches(&key_info),
+            "The backup info should match to the decryption key"
```

Comment:
nit: `"The backup info should match the decryption key"`

Reply:
https://github.com/matrix-org/matrix-rust-sdk/pull/2818/commits/f496e0f586e81c732e409963bd67a70de14f1b52